subsys/mpsl/Kconfig

```diff
@@ -25,6 +25,107 @@ config MPSL_TIMESLOT_SESSION_COUNT
 	help
 	  Maximum number of timeslot sessions.
 
+DT_COMPAT_NORDIC_NRF21540_GPIO := nordic,nrf21540-fem
+DT_COMPAT_SKYWORKS_SKY66112_11 := skyworks,sky66112-11
+
+config MPSL_FEM_NRF21540_GPIO_SUPPORT
+	bool
+	default $(dt_nodelabel_has_compat,nrf_radio_fem,$(DT_COMPAT_NORDIC_NRF21540_GPIO)) && !BT
+
+config MPSL_FEM_SKY66112_11_SUPPORT
+	bool
+	default $(dt_nodelabel_has_compat,nrf_radio_fem,$(DT_COMPAT_SKYWORKS_SKY66112_11))
+
+config MPSL_FEM
+	bool "Radio front-end module (FEM) support"
+# MPSL FEM is not supported on 53 yet
+	depends on !(SOC_NRF5340_CPUAPP || SOC_NRF5340_CPUNET) && \
+	           (MPSL_FEM_NRF21540_GPIO_SUPPORT || MPSL_FEM_SKY66112_11_SUPPORT)
+	help
+	  Controls if front-end module (FEM) is to be configured and enabled
+	  when MPSL is initialized. Default type of FEM to use depends on which
+	  compatible nodes are in devicetree.
+
+if MPSL_FEM
+
+choice MPSL_FEM_CHOICE
+	prompt "Radio front-end module (FEM) type"
+
+config MPSL_FEM_NRF21540_GPIO
+	depends on MPSL_FEM_NRF21540_GPIO_SUPPORT
+	bool "nRF21540 front-end module in GPIO mode"
+	help
+	  Note: nRF21540 is unavailable when Bluetooth LE is enabled
+
+config MPSL_FEM_SKY66112_11
+	depends on MPSL_FEM_SKY66112_11_SUPPORT
+	bool "SKY66112-11 front-end module"
+
+endchoice	# MPSL_FEM_CHOICE
+
+if MPSL_FEM_NRF21540_GPIO
+
+config MPSL_FEM_NRF21540_GPIO_GPIOTE_TX_EN
+	int "GPIOTE channel number used to control TX_EN pin of the front-end module"
+	default 7
```

Comment:
This is a user configurable setting, so there should be a proper help text here to let the user understand what this is about and how it might relate to other settings.

The prompt for this setting seems a bit long, consider if can be shortened and instead provide additional info in the help text.

Reply:
I reworked prompts and added missing help parts in this file